vidarrtest-regression.json.in

```diff
@@ -135,9 +135,9 @@
             "umiQC.postDedupBamQC.markDuplicates_timeout": null,
             "umiQC.postDedupBamQC.metadata": {
                 "donor": "Xenomorph",
-                "group id": "Weyland-Yutani",
+                "group id": "WY",
                 "library design": "WY-1001",
-                "tissue origin": "LV-426",
+                "tissue origin": "LV",
```

Comment:
I suggest to change it back, no need to edit a fictional name (but by keeping it as it was we may preserve some cultural references, a nice Easter egg)